pkgs/development/libraries/libpfm/default.nix

```diff
@@ -9,11 +17,13 @@ stdenv.mkDerivation rec {
     sha256 = "0jabhjx77yppr7x38bkfww6n2a480gj62rw0qp7prhdmg19mf766";
   };
 
-  installFlags = "DESTDIR=\${out} PREFIX= LDCONFIG=true";
+  makeFlags = lib.optional (arch != null) "ARCH=${arch}";
+
+  installFlags = [ "DESTDIR=\${out}" "PREFIX=" "LDCONFIG=true" ];
```

Comment:
It's usually a good idea to put these in makeFlags in case PREFIX/DESTDIR are hardcoded into source code somewhere.